src/modules/Packages.rb

```diff
@@ -1367,18 +1360,12 @@ def FindAndCopySlideDir(our_slidedir, source, search_for_dir, lang_long,
 
       if !imagesdir.nil?
         # where images should be cached
-        our_imagesdir = Builtins.sformat("%1/pic/", our_slidedir)
-        WFM.Execute(
-          path(".local.bash"),
-          Builtins.sformat("mkdir -p '%1'", String.Quote(our_imagesdir))
-        )
+        our_imagesdir = File.join(our_slidedir, "pic/")
+        ::FileUtils.mkdir_p(our_imagesdir)
 
-        copy_command = Builtins.sformat(
-          "cp -r '%1/%2/pic'/* '%3'",
-          String.Quote(imagesdir),
-          String.Quote(search_for_dir),
-          String.Quote(our_imagesdir)
-        )
+        source_path = File.join(imagesdir, search_for_dir, "pic", "*")
+
+        copy_command = "/usr/bin/cp -r #{source_path.shellescape} #{our_imagesdir.shellescape}"
```

Comment:
NP: Use [::FileUtils.cp_r](https://ruby-doc.org/stdlib-2.5.1/libdoc/fileutils/rdoc/FileUtils.html#method-c-cp_r) here?

Reply:
if we are sure that it cannot fail, then it is ok for me to change it.

Reply:
well, this won't work with fileutils as it uses "*" expansion in path. So I keep it as it is.